### ISSUE TITLE
Pokemon IVs changed to 0/0/0/0/0/0

### ISSUE THREAD

**Issue by st753m** [Thutmose/Pokecube#182]
#### Issue Description:

The IVs of 2 of my 3 often-used pokemon have vanished, replaced with all 0's.  

Specifically, it happened to my Grovyle and Magnemite.  I'm not sure how long its been going on, since I hadn't checked before.  I know for a fact that my grovyle (then treecko) had non-zero IVs when I checked it at the beginning of the game (they weren't great, mostly single-digit, but not 0). So far, it has not happened to my Xatu.

The EVs seem unaffected, or at least the EVs are about what I would expect from level 30 pokemon.  

#### Steps to reproduce:

Unknown.  I'll keep track of Xatu and see if anything happens to it.

____
#### Affected Versions (Do *not* use "latest"):

- Pokecube Core: 4.22.7
- Pokecube Revival (if applicable): 4.14.2
- Pokecube Mobs (if applicable): 2.4.8
- Minecraft: 1.10.2
- Forge: 12.18.3.2281

**Comment by Thutmose** [comment]
did this happen to both at the same time? could it have happened after a version update? I recently made some changes to how IVs were saved, and it may have somehow affected that?

**Comment by st753m** [comment]
I didn't check often enough to tell when; all I know is that it happened between level 8 and 28.  I did get magnemite for my party before natu, so its possible it happened very early on.  This was a new world, only played for 2 irl days.  

Because I forgot earlier, I'm running Pokecube with other mods: Botania, Ruins, Spice of Life, Harvestcraft, and Cooking For Blockheads.  These probably wouldn't mess with Pokecube, or IVs in particular, but maybe it's important to mention?

**Comment by st753m** [comment]
I decided to do some more tests.  I found that the problem is that my pokemobs change IVs randomly when they enter or leave their pokecube.  To test, I threw a pokemon on the ground and recalled it over and over (5 or 6 times, it doesn't happen every time) and then checked its IVs again.  

I tried this with just Thutcore and the 3 pokecube mods installed, and it still happened.  Just in case it was with my config settings I created a new profile, and it happened there too.  

I'm not sure how I ended up getting all 0's on 2 pokemobs this way.  Maybe whatever's randomizing them doesn't re-roll 0's, so 0/0/0/0/0/0 is the logical conclusion after several hundred times?

Edit: 0's do get re-rolled.
